### ISSUE TITLE
Click outline jumps to document top

### ISSUE THREAD

**Issue by gysabc** [typora/typora-issues#5948]
- [ ] Searched existing issues to avoid creating duplicates.

- [ ] Confirmed that it can be reproduced in built-in themes without customized css.

https://github.com/typora/typora-issues/assets/68911920/f7bcb336-6e51-4be2-b132-771fef25eb15

   If it only exists in 3rd party themes or css, you can still report it, but please attach the theme target or the css file. We may not "fix" it, if it is caused by 3rd party themes or css styles that we cannot support.

- [ ] Searched http://support.typora.io/

----

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots / Screencasts**
If applicable, add screenshots / screencasts to help explain your problem.

**Sample Markdown File**
Please attach or send us <hi@typora.io> the sample md file that may reproduce this.

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10 32bit]

**Typora Version**
e.g: 0.10.11

**Additional context**

**Comment by abnerlee** [comment]
Possible to send us <hi@typora.io> or attach the sample md file for us it check it?

**Comment by gysabc** [comment]
yes,of course. in the md file, all the titles are normal, except the title "4.2.2构建CAM". when click this title, it just jump to the beginning of the file






发件人：AbnerLee ***@***.***>
发送日期：2024-01-23 12:26:14
收件人：typora/typora-issues ***@***.***>
抄送人：Gao Yan ***@***.***>,Author ***@***.***>
主题：Re: [typora/typora-issues] Click outline jumps to document top (Issue #5948)

Possible to send us the sample md file for us it check it?

**Comment by gysabc** [comment]
This problem occurs sometimes, not always






发件人：AbnerLee ***@***.***>
发送日期：2024-01-23 12:26:14
收件人：typora/typora-issues ***@***.***>
抄送人：Gao Yan ***@***.***>,Author ***@***.***>
主题：Re: [typora/typora-issues] Click outline jumps to document top (Issue #5948)

Possible to send us the sample md file for us it check it?

**Comment by liushidexing** [comment]
I ran into the same problem, has this been resolved yet?
